spec/clinic.v1.yaml

```diff
@@ -2962,6 +2996,10 @@ components:
           readOnly: true
         summary:
           $ref: '#/components/schemas/PatientSummary'
+        lastReviewed:
```

Comment:
This two new fields ought to be an array

Reply:
done